crates/solver/src/solver/baseline_solver.rs

```diff
@@ -38,6 +40,9 @@ impl Solver for BaselineSolver {
         Ok(self.solve_(orders, liquidity, slippage))
     }
 
+    // We don't care about callbacks
+    fn notify_auction_result(&self, _auction_id: AuctionId, _result: AuctionResult) {}
```

Comment:
You could make this a default implementation in the trait. I think that is slightly better not haven't thought about it in detail.

Reply:
Fixed as I don't see any cons at this time.